UnleashedRecomp/hid/driver/sdl_hid.cpp

```diff
@@ -311,12 +327,15 @@ void hid::Init()
     SDL_SetHint(SDL_HINT_JOYSTICK_HIDAPI_WII, "1");
     SDL_SetHint(SDL_HINT_XINPUT_ENABLED, "1");
 
+    SDL_SetHint(SDL_HINT_GAMECONTROLLER_USE_BUTTON_LABELS, "0"); // Uses Button Labels. This hint is disabled for Nintendo Controllers.
+
```

Comment:
Remove this as it'll be handled by https://github.com/hedge-dev/UnleashedRecomp/pull/1045 instead.